.github/workflows/version.yml

```diff
@@ -5,22 +5,17 @@ on:
     tags:
       - '*.*.*'
 
-permissions: {}
+permissions:
```

Comment:
Nit: I'd recommend allowing a trigger on workflow dispatch useful for manual testing or redriving

Reply:
Added an option.